search_data.json

```diff
@@ -0,0 +1,50 @@
+---
+layout: null
+---
+{
+  {% for item_hash in data.resource %}
+    {% assign object = item_hash[1] %}
+    {% assign id = item_hash[0] %}
+    {% assign collection_id = object.uri | split:'/' | last %}
+    {% assign subjects = "" | split: ',' %}
+    {% assign agents = "" | split: ',' %}
+    {% assign dates = "" | split: ',' %}
+
+    {% for date in object.dates %}
+      {% assign dates = dates | push: date.expression %}
+    {% endfor %}
```

Comment:
In lieu of a date facet or ability to filter by date, I wonder about the utility of indexing date data?

Reply:
That's a good point. I was including the dates because they are technically the publication dates. But I'm happy to remove them since we don't have a date filter.